src/reconcile/rec_write.c

```diff
@@ -2221,28 +2221,34 @@ __wti_rec_pack_delta_internal(
 }
 
 /*
- * __rec_pack_delta_leaf --
- *     Pack a delta for a leaf page
+ * __rec_pack_delta_leaf_custom --
+ *     Pack a delta key and a delta value for a leaf page.
  */
 static int
-__rec_pack_delta_leaf(WT_SESSION_IMPL *session, WTI_RECONCILE *r, WT_SAVE_UPD *supd)
+__rec_pack_delta_leaf_custom(WT_SESSION_IMPL *session, WTI_RECONCILE *r, WT_SAVE_UPD *supd)
```

Comment:
What does custom mean?

Reply:
Custom means the value structure is different but I guess it's not really needed in the function name.